test-cases/terraform/aws/networking/default_sg_in_new_vpc/results_summary.json

```diff
@@ -1,7 +1,7 @@
 {
   "checkov": "yes",
   "cloudrail": "yes",
-  "kics": "no",
+  "kics": "yes",
```

Comment:
@kaplanlior and this

Reply:
KICS says: Instance With No VPC

Reply:
The case talks about the default security group

Reply:
Checkov: `Check: CKV2_AWS_12: "Ensure the default security group of every VPC restricts all traffic"`
Cloudrail: `Rule: Ensure all used default security groups of every VPC restrict all traffic`